library/network_connections.py

```diff
@@ -42,6 +42,17 @@
 
 DEFAULT_ACTIVATION_TIMEOUT = 90
 
+# Valid keys for the BONDING_OPTS parameter on a Bond interface
+VALID_BONDING_OPTS = [
```

Comment:
This is probably not necessary, since the ArgumentValidator takes care of ensuring that only valid options are provided - as far as I remember. Even if not, it contains the list of valid options already, so that could be re-used. What do you think?

Reply:
Indeed - this isn't about validation at this stage, it's just a list that I can iterate over in line 827-829 (of my branch) rather than building each option one by one as the existing code did.

I'll look into pulling this from ArgumentValidator.